ref(documentation): Update getting started doc (#728)
getsentry/sentry-relay: docs/index.md

information and improve event response time. It acts as a middle layer between
your application and Sentry.io.

**Note: Relay is still work in progress. The default Relay mode is at the moment only supported 

> for on-premises installations and beta testers. If you are using Relay to connect to Sentry and 

Reviewer: ```suggestion
for on-premise installations and beta testers. If you are using Relay to connect to Sentry and 
```

are not a beta tester you need to use `proxy` or `static` mode.
 

> If you want to beta test the `managed` mode while connecting to Sentry please get in 

Reviewer: ```suggestion
If you want to beta test the `managed` mode while connecting to Sentry, please get in 
```

you need to restrict all HTTP communication to a custom domain name, Relay can
act as an opaque proxy that reliably forwards events to Sentry.

> # Getting Started

Reviewer: Let's make this second level again, as it should be lower than the top-level heading.

```suggestion
## Getting Started
```

init` command. The command puts configuration files in the `.relay` folder
under the current working directory:

We will be creating a custom configuration since the default configuration creates a 

> `managed` mode configuration which is not supported by sentry at the current time.

Reviewer: This paragraph can be removed, as we're creating a default configuration after all. The primary audience of this document are customers with access to managed mode.

Author: yep

❯ docker run -v $(pwd)/configs/:/work/.relay/ getsentry/relay run
```

> This example command assumes that Relay's configuration (`config.yml` and `credentials.json`) are stored in `./configs/` 
> directory on the host machine.

Reviewer: ```suggestion
This example command assumes that Relay's configuration files, `config.yml` and
`credentials.json`, are stored in `./configs/` directory on the host machine.
```

> ## Logging and healthcheck

Reviewer: ```suggestion
## Logging and Healthcheck
```

> Now you have a running relay, you might have noticed that relay displays some `INFO` messages,

Reviewer: ```suggestion
Now you have a running Relay, you might have noticed that Relay displays `INFO` log messages,
```

including:
```sh
INFO  relay::setup >   log level: INFO
```

> This is the default logging level you can change this to show more or less info. 

Reviewer: ```suggestion
This is the default logging level and you can change this to show more or less info. 
```

For details about configuring logging please see _[logging]_ on the options page.

> Relay provides two urls for health check and monitoring the live status:
> 
> In our example relay is running at "http://localhost:3000" so the following urls are
> going to return a status check JSON file.
> 
> The `GET http://localhost:3000/api/relay/healthcheck/live/` is used to test if Relay
> is running and able to respond to http requests.
> 
> The `GET http://localhost:3000/api/relay/healthcheck/ready/` can be used to check if 
> Relay is running and has been able to authenticate with the upstream.
> 
> In case of success both endpoints return:

Reviewer: ```suggestion
Relay provides two URLs for checking liveness and readyness of the service:

- `GET /api/relay/healthcheck/live/`: Tests if Relay is running and listening to HTTP requests.
- `GET /api/relay/healthcheck/ready/`: Tests if Relay is authenticated with the upstream and
  operating normally.

In case of success, both endpoints return a _200 OK_ response:
```
